pkg/cmd/cli/cmd/rollback.go

```diff
@@ -57,14 +60,14 @@ func NewCmdRollback(fullName string, f *clientcmd.Factory, out io.Writer) *cobra
 	}
 
 	cmd := &cobra.Command{
-		Use:     "rollback DEPLOYMENT",
+		Use:     "rollback [dc/]config-name",
```

Comment:
We can never change the syntax of this command? What is the proposed syntax that gets us to a place where we can accept a config? `rollback --to-config=config --to-version=N` (no arg[0]?)